src/components/__tests__/checker.js

```diff
@@ -184,6 +186,35 @@ describe("check", () => {
       }).not.toThrow()
     })
   })
+
+  describe("close", () => {
+    beforeEach(() => jest.useFakeTimers())
+    afterEach(() => {
+      jest.useRealTimers()
+      jest.restoreAllMocks()
+      component.unmount()
+    })
+
+    it("calls editor.on('Remove') when mounted", () => {
+      component = mount(<Checker getBody={() => node} editor={fakeEditor} />)
+      instance = component.instance()
+      instance.check() // open it
+      jest.runAllTimers()
+      expect(fakeEditor.on).toHaveBeenCalled()
+      expect(fakeEditor.on).toHaveBeenCalledWith("Remove")
```

Comment:
Let's go back to this :)
```suggestion
      expect(fakeEditor.on.mock.calls[0][0]).toEqual("Remove")
```